portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -100,27 +106,35 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentsEntity.setProperty("email", email);
     commentsEntity.setProperty("message", comment);
     commentsEntity.setProperty("timestamp", timestamp);
+    commentsEntity.setProperty("userId", userId);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentsEntity);
     
-    // Redirect back to the HTML page.
-    response.sendRedirect("/index.html");
+    // Redirect back to the jsp page.
```

Comment:
You've remembered to update your comment this time, but comments like these that talk about specific implementation details (e.g., the language chosen to write the page at your redirect destination) tend to get stale. Better would be to say something more general about the intentions of the code, e.g., "Redirect back to the home page".

Reply:
This makes perfect sense, will change this, thanks!!